src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -70,6 +74,11 @@ public void execute(Ui ui, AccountList account) {
             ui.printMessage(ErrorMessage.INVALID_AMOUNT_TO_ADD);
         } catch (NullPointerException e) {
             ui.printMessage(ErrorMessage.NO_AMOUNT_PROVIDED);
+        } catch (NotEnoughInAccountException e) {
+            // this should not happen since we are adding money
+            ui.printMessage(ErrorMessage.NOT_ENOUGH_IN_ACCOUNT);
+        } catch (InvalidUpdateBalanceActionException e) {
```

Comment:
Do you want to remove this error?

Reply:
It is thrown by the function so unless we separate into different functions or the function prints on its own it cannot be helped